Refactor billing docs and update for streamlined compute billing

anataliocs opened the pull request:
**Description:**

1. Add Compute Billing and Storage Billing sections
2. Add Topup Compute funds page
3. Add View Compute usage page
4. Add Manage Storage plan page
5. Move Storage billing page from /storage to /billing/storage
6. Move Add-ons page to  /billing/storage

Issue:  #142


**Walkthrough of changes:**
![compute-plan-changes](https://github.com/spheronFdn/docs/assets/4750426/6ccd800e-12ad-4872-bef0-5c21a78a9b37)


**Updated Navigation for Compute vs Storage Billing:**
![image](https://github.com/spheronFdn/docs/assets/4750426/50a61d55-a9c1-4b3e-ac78-3c0fc5d84416)

anataliocs commented on pnpm-lock.yaml:
```diff
@@ -2169,4 +2169,4 @@ packages:
   /zwitch/2.0.4:
     resolution: {integrity: sha512-bXE4cR/kVZhKZX/RjPEflHaKVhUVl85noU3v6b8apfQEc1x4A+zBxjZ4lN8LqGd6WZ3dl98pY4o717VFmoPp+A==}
-    dev: false
+    dev: false
```
```suggestion
    dev: false
    
```

rekpero commented on pages/billing/compute/compute-usage.mdx:
```diff
@@ -0,0 +1,20 @@
+
+    Setting an automatic top up is the best way to ensure uninterrupted service for all your clusters.
+</Callout>
+
```
mention this -> when you go inside the cluster, you can even see full breakdown on each instance usage and monthly cost.

rekpero commented on pages/billing/compute/topup-compute-funds.mdx:
```diff
@@ -0,0 +1,34 @@
+Spheron Compute uses a usage-based billing model.  Users can top up their balance using either a credit card or a web3 wallet, which are used to pay for compute resources.
+
+To Top Up your Compute balance you first need to either:
+- Add a [Web3 Payment Method](/billing/payment-method/#connect-wallet) and deposit funds in your [Spheron escrow wallet](/billing/funds/)
```
instead of aggregating it like this, I think it would be better if you do 2 guide - 
1. Topup with your fav web3 wallet
2. Topup with your credit card

Then show the whole process of attaching the wallet, depositing and then paying for the topup. 

Reason -> A Lot of users think that depositing the token is the last step of the top-up and their instances are going down because they don't have any balance to pay for it.

anataliocs replied:
Ahhh thank you this is a great call out.  Will more explicitly define those two paths.

rekpero commented on pages/billing/storage/storage-plan.mdx:
```diff
@@ -0,0 +1,34 @@
+1. Amount of available total Storage(GB)
+2. Amount of available total Bandwidth(GB)
+
+The Free Starter Plan is capped at 5GB of IPFS storage and 10GB of bandwidth.  Arweave is not available for Starter Plan users.
```
make this a note instead

rekpero commented on pages/billing/storage/storage-plan.mdx:
```diff
@@ -0,0 +1,34 @@
+
+## Upgrading your Storage Plan
+
+When you need more storage, bandwidth or access to Arweave you can upgrade your plan.
```
access to arweave doesn't particularly need any upgrade, user can buy arweave storage ad-on to access it

rekpero commented on pages/billing/storage/storage-plan-details.mdx:
```diff
@@ -0,0 +1,38 @@
+
+| **Ad-ons**        | **Price**    |
+| ----------------- | ------------ |
+| Parallel Uploads  | $12 / upload |
```
```suggestion
| Parallel Uploads  | $2 / upload |
```

rekpero commented on pages/billing/funds.mdx:
```diff
@@ -1,29 +1,34 @@
+**Follow these steps to deposit funds:**
+1. Connect your Web3 Wallet, if it is not already connected.
+2. Select **Deposit Funds** from the **Payment Method** dropdown.
 3. Add the amount you want to approve and Click "Approve."
 4. Add the amount you want to deposit and Click "Deposit."
```
we clubbed these 2 section, now approve and deposit tx are chained

rekpero commented on pages/billing/_meta.json:
```diff
@@ -1,8 +1,9 @@
-  "add-on": "Add-on",
-  "invoices": "Invoices & Usages",
   "credits": "Credits",
+  "invoices": "Invoices & Usages",
```
can you add the payment history in the billing section as well, you can find it in the app

anataliocs replied:
Will handle in new PR:  [Add-Payment-History-page-for-Compute-Storage](https://www.notion.so/sphn/Add-Payment-History-page-for-Compute-Storage-b172d3ccca3649d39d850f5d40fdeded?pvs=4)

rekpero reviewed (changes requested):
@anataliocs review done, check my RC

anataliocs commented:
Need to add: 

Mitra:
> can you update the main page to include storage and compute billing? like a quick intro and then 2 blocks like we have for upgrade to pro to send them to compute and storage section

anataliocs commented:
Compute and storage sections on root billing page added:

![image](https://github.com/spheronFdn/docs/assets/4750426/7328d726-33a4-4129-a0cc-36633bcd8741)

anataliocs commented:
Added Storage and compute topup elements and added to billing root page:
![image](https://github.com/spheronFdn/docs/assets/4750426/3b58c72f-cc79-4506-831d-44e06d647bac)

anataliocs commented:
Added Metamask SVG

![image](https://github.com/spheronFdn/docs/assets/4750426/71100bcc-ab1d-4fe8-bd21-cec5fee3d360)

rekpero commented on pages/billing.mdx:
```diff
@@ -10,6 +12,18 @@ If you are not already a pro user, you can upgrade to a pro plan to enjoy additi
 
 <Upgrade />
 
+## Top Up Compute Balance
```
put this above upgrade to Pro, since it is for site billing, move to below storage section

anataliocs replied:
Definitely, that makes sense!

anataliocs replied:
Is this the layout we want?
![image](https://github.com/spheronFdn/docs/assets/4750426/dda60101-3453-419d-8b80-5b0bf2fabd0e)

rekpero replied:
yes, this is good

rekpero commented on components/storage-guide.tsx:
```diff
@@ -0,0 +1,32 @@
+
+const STORAGE_LIST = [
+    {
+        path: "/billing/storage/storage-plan/",
```
Add available storage plan in the list to make it consistent even tho it's not needed but it's a part of the storage billing so it make sense to have this

anataliocs replied:
Yea, it looks better when it's symmetrical and it makes sense to have easy access to the page in this context

rekpero reviewed (changes requested):
last bit of changes needed and we are good to go @anataliocs really great job

anataliocs commented:
Also tested mobile responsive view locally:

![image](https://github.com/spheronFdn/docs/assets/4750426/a9a90eee-cdb2-4c72-aea5-aef78c3e217b)
